### ISSUE TITLE
[hdfilme.cx] Ads randomly appearing during site navigation.

### ISSUE THREAD

**Issue by gettysburg** [easylist/easylist#9993]
### List the website(s) you're having issues:

```
https://hdfilme.cx
https://hdfilme.cx/filme1/james-bond-007-keine-zeit-zu-sterben-17824-stream
```

### What happens?

During normal site navigation, advertisements randomly appear on the bottom right corner, they usually look like this:

![image](https://user-images.githubusercontent.com/2256003/145473571-acc5b286-534c-47ba-8338-f9c6715dbb31.png)

Using the uBlock Origin content picker, I was able to create the following filter which seems to get rid of the advertisements for now:

```##iframe:nth-of-type(3)```

### List Subscriptions you're using:

Every single one available in the latest developer build of uBlock Origin, except for the region-related ones, where I only enable EasyList Germany.

### Your settings

- OS/version: Windows Server 2019 LTSC / 1809 17763.2300
- Browser/version: Firefox 91.4.0 ESR
- Adblock Extension/version: uBlock Origin 1.39.3b6
- Additional Extensions: 

1.) HTTPS Everywhere in _blocking_ mode.
2.) An extension to strip the DNT header from all requests.
3.) GreaseMonkey.

Noteworthy browser settings:

```
javascript.options.asmjs = false
javascript.options.baselinejit = false
javascript.options.ion = false
javascript.options.native_regexp = false
javascript.options.wasm = false

privacy.resistFingerprinting = true
```

.. and many more security / hardening related changes, Firefox' built in "Enhanced Tracking Protection" is set to 'Strict' as well.

### Other details:

The reason for creating this bug report was me assuming that the **element picker** of uBlock Origin was **broken** in a way that made me unable to select the correct element, when trying to create a filter - this was the case in the past, i.e. I hovered over the ad and uBlock Origin suggested a bogus filter (a valid ReCaptcha element), instead of the one listed above. Now, after trying again _while writing this bug report_, and with an upgraded developer version of uBlock Origin, I notice that this bug seems to have been fixed, making the report semi-obsolete - of course, the site and the filter should still be added to EasyList, or rather EasyList Germany, given that this is an entirely German site with no planned support for English speaking users.

Can someone else please confirm that the filter above is enough to completely stop the advertisements in the bottom, right corner?

I also, in the past, created two additional filters:

```
hdfilme.cx###fv-wrapper
hdfilme.cx##.blinking.alert
```

A quick navigation around the site and a look at uBlock Origin's logger confirms that both filters are still needed to remove annoyances and thus should be added to the appropriate filter lists. 

There is also an invisible overlay on the site, which tries to open a pop-up, meaning you can't just press play, the mouse event gets caught by some third-party handler, and only after 1-2 clicks on the play button, it actually registers - due to limited time I couldn't make a filter for this yet.

Thank you for your time.

**Comment by spirillen** [comment]
Can't reproduce it

**Comment by gettysburg** [comment]
@spirillen

My guess is that they either set a very low random chance for ads to appear, or only distribute them to users of certain countries based on IP-address or by utilizing the Accept-Language header, given that the ads themself link to news articles about German sport events (however, I have also seen an advertisement for presumably a dating site, that very ad has a purple background with an anime character in the middle of it, and usually, but not exclusively, appears if you close the two ads shown above, which is possible by hovering over them, an X will appear on the right side of each ad).

Even for me (essentially their target demographic), and before I added the filter above, it sometimes took 15-30 minutes of site navigation before anything happened. 

I have just watched a submarine movie on that very site and did not get any ads during the 2 hour period, and the proposed filter from above was triggered on page load.

Someone will have to dig into their advertising networks code to find out what exactly causes them to be displayed (or not).

**Comment by spirillen** [comment]
Hey @gettysburg 

That is the very reason I posted my comment (Used tor-browser) 

By my comment two things can happen, one the rules get a rewrite or those with the merge keys now knows any of you mentioned reasons is in play. :wink: :wave:

**Comment by gettysburg** [comment]
Dear list maintainers,

I have confirmed that the first filter from my initial post gets rid of all three types of ads on this site.

Can we please get it merged into EasyList or EasyList Germany?
